From 5e48deb25e8c1f2ee596120ed5bcbdb6daf3dee7 Mon Sep 17 00:00:00 2001
From: Alex Wilson <alex.wilson@joyent.com>
Date: Fri, 22 Jun 2018 18:59:32 -0700
Subject: [PATCH] MANTA-3787 waferlock should look for "zookeeper" as well as
 "binder" Reviewed by: Cody Peter Mello <cody.mello@joyent.com> Approved by:
 Cody Peter Mello <cody.mello@joyent.com>

---
 sapi_manifests/waferlock/template | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/sapi_manifests/waferlock/template b/sapi_manifests/waferlock/template
index 3b61b52..7609bf9 100644
--- a/sapi_manifests/waferlock/template
+++ b/sapi_manifests/waferlock/template
@@ -26,7 +26,9 @@
 			"{{{moray_domain}}}",
 			"{{{BINDER_SERVICE}}}",
 			{{! Don't use pgstatsmon_domain, might not be set. }}
-			"pgstatsmon.{{{datacenter_name}}}.{{{dns_domain}}}"
+			"pgstatsmon.{{{datacenter_name}}}.{{{dns_domain}}}",
+			{{! Same deal here: this is for legacy DCs. }}
+			"zookeeper.{{{datacenter_name}}}.{{{dns_domain}}}"
 		],
 		"sapi_url": "{{{sapi-url}}}",
 		"sapi_services": ["sdc/moray"],
-- 
2.21.0

